### PR TITLE
Initial 2022 home page

### PR DESCRIPTION
**Pull request by osg-cat**

### CODE REVIEW

**Comment by osg-cat** [comment]
I’d like this version to get posted, although we are still refining the content. I highly suspect there will be an update yet today or tomorrow.

File: mkdocs.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ theme:
 
 nav:
   - Home: index.md
+  - 2022:
+    - General Info: 2022/index.md
```
Comment (brianhlin) [inline mkdocs.yml:11]: Just an FYI: you can skip updating `mkdocs.yml` to generate the new page but hide it from the ToC if you want to share a preview of a page internally.

**Review by brianhlin — APPROVED** [review]
LGTM, CI failure unrelated.

File: mkdocs.yml (hunk 1 above)
Comment (osg-cat) [reply mkdocs.yml:11]: Ah, interesting, thanks. I don’t really care if anyone sees this before it’s 100% done … and no one will find it by accident, I think.

**Pull request merged**